catalogue/webapp/components/PhysicalItemDetails/PhysicalItemDetails.tsx

```diff
@@ -0,0 +1,113 @@
+import { FunctionComponent } from 'react';
+import styled from 'styled-components';
+import ButtonInlineLink from '@weco/common/views/components/ButtonInlineLink/ButtonInlineLink';
+import Space from '@weco/common/views/components/styled/Space';
+import { classNames, font } from '@weco/common/utils/classnames';
+
+const Wrapper = styled.div`
+  border-bottom: 1px solid ${props => props.theme.color('pumice')};
+  margin-bottom: 20px;
+`;
+
+const Row = styled(Space).attrs({
+  v: { size: 'm', properties: ['margin-bottom'] },
+})``;
+
+const Grid = styled.div`
+  ${props => props.theme.media.medium`
+    display: grid;
+    grid-template-columns: 200px 150px 125px;
+    grid-column-gap: 30px;
```

Comment:
should these kind of values be part of the theme do you think?

Reply:
Possibly eventually. But it's all a bit up in the air design-wise at the moment and seemed like it might be limiting to get too strict too soon with it.